althea_kernel_interface/src/lib.rs

```diff
@@ -545,7 +548,7 @@ Bridge chain: INPUT, entries: 3, policy: ACCEPT
         };
 
         match ki.delete_flow_counter_linux(
-            "0:0:0:aa:0:2".parse::<HwAddr>().unwrap(),
+            MacAddress::parse_str("00:00:00:aa:00:02").unwrap(),
```

Comment:
Hmm, surprised that `.parse` doesn't work with this library, I think it just needs the `ToString` trait IIRC. No big deal I guess...

Reply:
Sure, I'm all in for alternatives. After looking at your other comments this lib doesn't seem as good as before

Reply:
The only real issue I have with this lib is the less robust parsing discussed in the comment above